franco/Project/org.fundacionjala.automationproject/src/test/java/org/fundacionjala/automationproject/steps/BillingAddressSteps.java

```diff
@@ -0,0 +1,57 @@
+package org.fundacionjala.automationproject.steps;
+
+import static org.fundacionjala.automationproject.entities.AddressInfo.ADDRESS;
+import static org.fundacionjala.automationproject.entities.AddressInfo.CITY;
+import static org.fundacionjala.automationproject.entities.AddressInfo.COUNTRY;
+import static org.fundacionjala.automationproject.entities.AddressInfo.STATE;
+import static org.fundacionjala.automationproject.entities.AddressInfo.ZIP;
+import static org.junit.Assert.assertTrue;
+
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.fundacionjala.automationproject.entities.AddressInfo;
+import org.fundacionjala.automationproject.ui.PurchasePage;
+
+/**
+ * Created by Franco Aldunate on 11/22/2017.
+ */
+public class BillingAddressSteps {
+  private final PurchasePage purchasePage;
+  private final AddressInfo addressInfo;
+
+  public BillingAddressSteps(AddressInfo addressInfo, PurchasePage purchasePage) {
+    this.addressInfo = addressInfo;
+    this.purchasePage = purchasePage;
+  }
+
+  /**
+   * <p>This method fills the billing address information.</p>
+   *
+   * @param address  is the given billing address.
+   * @param aCity    is the given billing address city.
+   * @param aState   is the given billing address state.
+   * @param aZip     is the given billing address postal code.
+   * @param aCountry is the given billing address country.
+   */
+  @When("^I fill billing address info with address \"([^\"]*)\", "
+    + "city \"([^\"]*)\", state \"([^\"]*)\", postal code \"([^\"]*)\""
+    + " and country \"([^\"]*)\"$")
+  public void iFillBillingAddressInfoWithAddressCityStatePostalCodeAndCountry(String address,
```

Comment:
I think that this method name is very long.
You can change it to,
fillBillingAddressInfo(String address , String  ........................)

Reply:
Thank you, I shortened this method to: iFillBillingAddressWithTheFollowingInformation(List<Address> addressList)